code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1001,6 +1001,12 @@
 			"<span class='userdanger'>[user] has attempted to [atk_verb] [target]!</span>", null, COMBAT_MESSAGE_RANGE)
 			return 0
 
+		var/teethhitcheck = rand(0,9)
+		if(istype(affecting, /obj/item/bodypart/head) && prob(teethhitcheck * (user.zone_selected == "mouth" ? 3 : 1))) //MUCH higher chance to knock out teeth if you aim for mouth
+			var/obj/item/bodypart/head/U = affecting
+			if(U.knock_out_teeth(get_dir(user, target), round(rand(28, 38) * ((teethhitcheck*2)/100))))
```

Comment:
There used to be a hitcheck on punching, but I don't know where it is now so I just added that instead.